package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "apttus.ui.pcom.rc",
-  "version": "2303.0.0",
+  "version": "2402.0.0",
```

Comment:
Version shouldn't be updated.

Reply:
Doubt: since our sdk versions are updated to 2402.X.X shouldn't this version also be in sync with the others?